packages/api/src/jobs/upload_content.ts

```diff
@@ -4,7 +4,7 @@ import { logger } from '../utils/logger'
 import { htmlToHighlightedMarkdown, htmlToMarkdown } from '../utils/parser'
 import { isFileExists, uploadToBucket } from '../utils/uploads'
 
-export const UPLOAD_CONTENT_JOB = 'UPLOAD_CONTENT_JOB'
+export const UPLOAD_CONTENT_JOB = 'upload-content'
```

Comment:
with this name change, we might have a small issue with items in the queue during deploy.

Reply:
I wonder if I can pause the queue a little bit when all the `upload-content` jobs are done